scpdt/plugin.py

```diff
@@ -19,12 +23,38 @@ def pytest_configure(config):
     doctest.DoctestModule = DTModule
     doctest.DoctestTextfile = DTTextfile
 
+def pytest_unconfigure(config):
+    """
+    Delete all local files copied to the current working directory
+    """
+    if len(copied_files) > 0:
```

Comment:
Nit: `if copied_files` (works for None and empty containers)

Reply:
Ah, this is what a global `copied_files` is for. We should try avoiding globals indeed. If `config` is our DTConfig, it has the list of files (a dict of lists actually). If it's a pytest object, can we attach our config to it?

Reply:
> If it's a pytest object, can we attach our config to it?

It is a pytest object, I will look into alternative ways to achieve the same thing.